lib/widgets/description.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'chiplist.dart';
+import 'description_text.dart';
+import 'icon.dart';
+
+class Description extends StatelessWidget {
+  static const double dsSpaceBetweenChipList = 2;
+  static const double dsPadding = 32;
+  static const double dsSizedBoxHeight = 10;
+  static const double dsVoteAvgFontSize = 15.0;
+  static const double dsChipListFontSize = 10.0;
+  static const int numberOfFullStars = 3;
+  static const int numberOfHalfStars = 1;
+  static const int numberOfBorderStars = 1;
```

Comment:
These numbers should be dynamically computed, right? Depending on the `voteAverage`

Reply:
solved